src/main/scala/gitbucket/notifications/service/NotificationsHook.scala

```diff
@@ -53,57 +53,55 @@ class IssueHook extends gitbucket.core.plugin.IssueHook
   with IssuesService {
 
   override def created(issue: Issue, r: RepositoryInfo)(implicit context: Context): Unit = {
-    Notifier().toNotify(
-      subject(issue, r),
-      message(issue.content getOrElse "", r)(content => s"""
-        |$content<br/>
-        |--<br/>
-        |<a href="${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}"}">View it on GitBucket</a>
-        """.stripMargin)
-    )(recipients(issue))
+    val markdown =
+      s"""|${issue.content getOrElse ""}
+          |
+          |----
+          |[View it on GitBucket](${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}"})
+          |""".stripMargin
+
+    Notifier().toNotify(subject(issue, r), markdown, Some(toHtml(markdown, r)))(recipients(issue))
   }
 
   override def addedComment(commentId: Int, content: String, issue: Issue, r: RepositoryInfo)(implicit context: Context): Unit = {
-    Notifier().toNotify(
-      subject(issue, r),
-      message(content, r)(content => s"""
-        |$content<br/>
-        |--<br/>
-        |<a href="${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}#comment-$commentId"}">View it on GitBucket</a>
-        """.stripMargin)
-    )(recipients(issue))
+    val markdown =
+      s"""|${toHtml(content, r)}
+          |
+          |----
+          |[View it on GitBucket](${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}#comment-$commentId"})
+          |""".stripMargin
+
+    Notifier().toNotify(subject(issue, r), markdown, Some(toHtml(markdown, r)))(recipients(issue))
   }
 
   override def closed(issue: Issue, r: RepositoryInfo)(implicit context: Context): Unit = {
-    Notifier().toNotify(
-      subject(issue, r),
-      message("close", r)(content => s"""
-        |$content <a href="${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}"}">#${issue.issueId}</a>
-        """.stripMargin)
-    )(recipients(issue))
+    val markdown =
+      s"""|close #[${issue.issueId}](${s"${context.baseUrl}/${r.owner}/${r.name}/issues/${issue.issueId}"})
```

Comment:
`#` symbol is dropped out of link text?

Reply:
HTML:
![2017-08-26 13 38 55](https://user-images.githubusercontent.com/1094760/29738665-17a0b2e0-8a64-11e7-990e-24830dcc4b1c.png)

Text:
![2017-08-26 13 39 05](https://user-images.githubusercontent.com/1094760/29738666-1da9ccb2-8a64-11e7-972a-bdd5a74cbf0e.png)